src/dotnet/commands/dotnet-sln/LocalizableStrings.resx

```diff
@@ -165,4 +165,7 @@
   <data name="InRoot" xml:space="preserve">
     <value>Place project in root of the solution, rather than creating a solution folder.</value>
   </data>
+  <data name="AddProjectSolutionFolderArgumentDescription" xml:space="preserve">
+    <value>The destination solution folder path to add the projects to</value>
```

Comment:
```suggestion
    <value>The destination solution folder path to add the projects to.</value>
```

Reply:
I've noticed the `xlf` auto-updated files aren't updated alongside the original `resx` file.. should that be done?

Reply:
Nevermind. I just found out.